.travis.yml

```diff
@@ -12,14 +12,15 @@ php:
     - 5.5
     - 5.6
     - 7.0
-    - hhvm
 
 matrix:
     include:
         - php: 5.3
-          env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
+          env: COMPOSER_FLAGS='--prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
this change does not make sense. This job is precisely about testing the lower bound